gangplank/internal/ocp/filer.go

```diff
@@ -319,11 +348,9 @@ func (m *minioServer) ensureBucketExists(ctx context.Context, bucket string) err
 		return err
 	}
 
-	be, err := mc.BucketExists(ctx, bucket)
-	if err != nil {
+	if be, err := mc.BucketExists(ctx, bucket); err != nil {
 		return err
-	}
-	if be {
+	} else if be {
```

Comment:
dumb question: does converting this from a separate if cause any problems? `be` will be properly defined from the `mc.BucketExists()` call?

Reply:
Yup. the scope of `be` is relative to the if/else if, and falls out of scope after.